src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsagePlugin/refresh-disk-usage.js

```diff
@@ -0,0 +1,11 @@
+function refreshDiskUsage(a) {
+
+    fetch("refresh", {
+        method: "POST",
+        headers: crumb.wrap({
+            "Content-Type": "application/x-www-form-urlencoded",
```

Comment:
This header wasn't in the old version, any reason for adding this?

Reply:
Not needed. It was used for testing, and I forgot to remove it (no change in behavior). Thanks!